src/pages/index.tsx

```diff
@@ -0,0 +1,51 @@
+import classNames from "classnames";
+import styles from "./index.module.css";
+
+const CODE_OF_CONDUCT_LINK =
+	"https://docs.google.com/document/d/e/2PACX-1vQf_7z4Ok-zyUY9vej4ZDafitByM6TOuZ8ju2rcEdEuKtuTYYrtDNjjw0EZRmt9pVQ2k7PqghYf16-F/pub";
+
+const HomePage = () => {
+	return (
+		<div className={styles.root}>
+			<header className={styles.header}>
+				{/* eslint-disable-next-line @next/next/no-img-element */}
+				<img
```

Comment:
Why not use Next's `Image` component here?

Reply:
I didn't want to have to define the `width` and `height` properties, but now remember the `next/image` way around that is by [statically importing the image file](https://nextjs.org/docs/app/building-your-application/optimizing/images#local-images) and passing it in through `src`!

Updated in https://github.com/React-Indy/website/pull/2/commits/f958b9b8ce96d24da08722d1e5c3bf0e70507fc9. Great callout 💯